transport/src/main/java/io/netty/channel/pool/FixedChannelPool.java

```diff
@@ -228,6 +228,11 @@ public void onTimeout(AcquireTask task) {
         this.maxConnections = maxConnections;
         this.maxPendingAcquires = maxPendingAcquires;
     }
+    
+    /** Returns the number of acquired channels that this pool thinks it has. */
+    public int getAcquiredChannelCount() {
+        return acquiredChannelCount;
```

Comment:
This is not safe as acquiredChannelCount is not atomic.

Reply:
I see three options, do you have a preference?

1) Make `acquiredChannelCount` atomic
2) Enforce the caller is calling this in the event loop
3) Return a future and call it in the event loop

Reply:
I guess making acquiredChannelCount atomic is good enough

Reply:
Made it atomic.